Common/UnitDefinitions/ForcePerLength.json

```diff
@@ -83,6 +83,18 @@
           "Abbreviations": [ "lbf/yd" ]
         }
       ]
+    },
+    {
+      "SingularName": "KilopoundForcePerFoot",
+      "PluralName": "KilopoundsForcePerFoot",
+      "FromUnitToBaseFunc": "x*14593.90292",
+      "FromBaseToUnitFunc": "x/14593.90292",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "kipf/ft" ]
```

Comment:
Isn't the abbreviation `kip/ft` ? Should we maybe add both, and if so, which should be the first array item used by ToString() by default?
Ref: https://www.cadforum.cz/cadforum_en/units.asp?u=forcePerLength

Reply:
Same goes for the now-merged https://github.com/angularsen/UnitsNet/pull/747.

Reply:
Another ref: https://en.wikipedia.org/wiki/Kip_(unit)

Reply:
Well to be honest when I started on the implementation for imperial support in our module, I never really understood what the correct abbreviation was or should be.
We are using the abbreviation "k/ft" in our module.
Some places mention kipf/ft, other kip/ft.

For KilopoundsForce for example, we are using simply "k" and not kipf or kip.
For all i care, we can add all the abbreviations, or add them afterwards if someone needs them

candidates:

1.) kip/ft
2.) kipf/ft
3.) k/ft

Reply:
Okay, mind if i update Force along too in this PR? or should I create a separate PR for that?

Reply:
I'll do it separately

Reply:
Done in #751